jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/tables/DataTable.java

```diff
@@ -249,7 +258,7 @@ public boolean hasGroup(String groupName) {
 
     @JDIAction("Check that {name} is loading")
     public boolean isLoading() {
-        return find(PROGRESS_BAR_LOCATOR).isExist();
+        return find(PROGRESS_BAR_LOCATOR).isExist() && find(PROGRESS_BAR_LOCATOR).attr("style").contains("4px");
     }
 
```

Comment:
1) We have interface 'IsLoading' and 'LoadingAssert', please implement it into the task.
2) for height we have interfaces 'HasMeasurement' and 'MeasurementAssert', please, implement it into the task.

Reply:
Refactored